src/EFCore.PG/Query/ExpressionTranslators/Internal/NpgsqlNetworkTranslator.cs

```diff
@@ -0,0 +1,148 @@
+                return new CustomBinaryExpression(expression.Arguments[1], expression.Arguments[2], "<=", typeof(bool));
+
+            case nameof(NpgsqlNetworkExtensions.Equal):
+                return new CustomBinaryExpression(expression.Arguments[1], expression.Arguments[2], "=", typeof(bool));
+
+            case nameof(NpgsqlNetworkExtensions.GreaterThanOrEqual):
+                return new CustomBinaryExpression(expression.Arguments[1], expression.Arguments[2], ">=", typeof(bool));
+
+            case nameof(NpgsqlNetworkExtensions.GreaterThan):
+                return new CustomBinaryExpression(expression.Arguments[1], expression.Arguments[2], ">", typeof(bool));
+
+            case nameof(NpgsqlNetworkExtensions.NotEqual):
+                return new CustomBinaryExpression(expression.Arguments[1], expression.Arguments[2], "<>", typeof(bool));
+
+            case nameof(NpgsqlNetworkExtensions.ContainedBy):
+                return new CustomBinaryExpression(expression.Arguments[1], expression.Arguments[2], "<<", typeof(bool));
+
+            case nameof(NpgsqlNetworkExtensions.ContainedByOrEqual):
+                return new CustomBinaryExpression(expression.Arguments[1], expression.Arguments[2], "<<=", typeof(bool));
+
+            case nameof(NpgsqlNetworkExtensions.Contains):
+                return new CustomBinaryExpression(expression.Arguments[1], expression.Arguments[2], ">>", typeof(bool));
+
+            case nameof(NpgsqlNetworkExtensions.ContainsOrEqual):
+                return new CustomBinaryExpression(expression.Arguments[1], expression.Arguments[2], ">>=", typeof(bool));
+
+            case nameof(NpgsqlNetworkExtensions.ContainsOrContainedBy):
+                return new CustomBinaryExpression(expression.Arguments[1], expression.Arguments[2], "&&", typeof(bool));
+
+            case nameof(NpgsqlNetworkExtensions.Not):
+                return new CustomUnaryExpression(expression.Arguments[1], "~", expression.Arguments[1].Type);
+
+            case nameof(NpgsqlNetworkExtensions.And):
+                return new CustomBinaryExpression(expression.Arguments[1], expression.Arguments[2], "&", expression.Arguments[1].Type);
+
+            case nameof(NpgsqlNetworkExtensions.Or):
+                return new CustomBinaryExpression(expression.Arguments[1], expression.Arguments[2], "|", expression.Arguments[1].Type);
+
+            case nameof(NpgsqlNetworkExtensions.Add):
+                return new CustomBinaryExpression(expression.Arguments[1], expression.Arguments[2], "+", expression.Arguments[1].Type);
+
+            case nameof(NpgsqlNetworkExtensions.Subtract):
+                return new CustomBinaryExpression(expression.Arguments[1], expression.Arguments[2], "-", expression.Arguments[1].Type);
+
+            case nameof(NpgsqlNetworkExtensions.Abbreviate):
+                return new PgFunctionExpression("abbrev", typeof(string), new[] { expression.Arguments[1] });
```

Comment:
Please use `SqlFunctionExpression` instead of `PgFunctionExpression`, the latter is there only for named arguments (which is a PostgreSQL-specific syntax) which isn't used here.